ibm_catalog.json

```diff
@@ -0,0 +1,249 @@
+{
+  "products": [
+    {
+      "label": "Cloud automation for Observability agents",
+      "name": "deploy-arch-ibm-observability-agents",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging_monitoring"
+      ],
+      "short_description": "Deploys IBM Cloud Observability agents to a cluster",
+      "long_description": "Solutions that support deploying IBM Cloud Observability agents for logging, monitoring.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/solutions/baseline/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-agents/main/images/observability-icon.svg",
+      "flavors": [
+        {
+          "label": "Baseline",
+          "name": "baseline",
+          "working_directory": "solutions/baseline",
+          "compliance": {},
+          "iam_permissions": [
+            {
+              "service_name": "containers-kubernetes",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Viewer"
+              ]
+            }
+          ],
+          "architecture": {
+            "features": [
+              {
+                "title": "Works with any Openshift or k8s cluster",
+                "description": "Works with Red Hat OpenShift, and Kubernetes clusters."
+              },
+              {
+                "title": "Deploys the IBM Cloud Monitoring agent on an existing cluster.",
+                "description": "Deploys the IBM Cloud Monitoring agent on an existing Red Hat OpenShift cluster."
+              },
+              {
+                "title": "Deploys the IBM Logs agent on an existing cluster.",
+                "description": "Deploys the IBM Logs agent on an existing Red Hat OpenShift cluster."
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-agents/refs/heads/addons/reference-architecture/deployable-architecture-observability-agents.svg",
+                  "caption": "Observability agents on a Red Hat OpenShift cluster",
+                  "type": "image/svg+xml"
+                },
+                "description": "Observability agents deployed on a Red Hat OpenShift cluster on a secure landing zone."
+              }
+            ]
+          },
+          "dependencies": [
+            {
+              "name": "testing-deploy-arch-ibm-observability",
+              "id": "daf2cf91-2384-4205-b7f4-621299a4896d",
```

Comment:
This is some random dev tile. Can you use what we have in the tile now please: https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/fc9c1881ea5535cba400960fcf5618db5bbfaba3/ibm_catalog.json#L529-L550